### DISCUSSION TITLE
Slack Integration not working

### DISCUSSION THREAD

**Discussion by bencohen4** [vercel/community#5233] [Bug Report]
### Page to Investigate

https://vercel.com/integrations/slack

### Steps to Reproduce

I was signed into 3 different Slack workspaces. Vercel was trying to make me connect Vercel to the first Slack workspace, but I wanted to connect it to #2. So I signed out of #1. But now if I try to connect it, it goes into an infinite loop of 

```
Re-authentication required
The Slack integration is no longer connected to your Slack workspace. Please re-authenticate to resume your configuration.
```
I click on `Authenticate with Slack`

Then I am shown a page detailing the permissions I'm giving Vercel, and I click `Allow`

Then it says `Vercel is requesting permission to access the {name} Slack workspace`

And this continues indefinitely.

**Comment by juanfel254** [comment]
I am experiencing the same issue. Any updates on how to fix this?

**Reply by eirikrsuzuki** [reply]
Vercel does not allow me to send messages to the app

**Comment by yerffejytnac** [comment]
Also experiencing Slack deployment notifications. Have tried unsubscribing/resubscribing/reauthenticating the integration. Seems to have broken recently.
